synapse/storage/schema/main/delta/65/12_update_sqlite_fts4_tokenizer.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
We'll need to update the branch and move this into `delta/70` folder

Reply:
It will need to go into `delta/71` since the schema version has been bumped.